src/mono/wasm/runtime/startup.ts

```diff
@@ -842,48 +880,62 @@ let configLoaded = false;
  * @param {string} configFilePath - relative path to the config file
  * @throws Will throw an error if the config file loading fails
  */
-export async function mono_wasm_load_config(configFilePath: string): Promise<void> {
+export async function mono_wasm_load_config(configFilePath?: string): Promise<void> {
     if (configLoaded) {
+        return afterConfigLoaded.promise;
```

Comment:
Does this need to do `await afterConfigLoaded.promise` instead?

Reply:
that's the same thing. The caller will await it. It doesn't matter which promise.